### DISCUSSION TITLE
Running `npx sb init` on a simple react project results in a broken Storybook

### DISCUSSION THREAD

**Discussion by Bikeman868** [storybookjs/storybook#13934] [Help]
Launching Storkbook displays a warning message, and the UI says that their are no stories configured.

1. Clone this repo https://github.com/Bikeman868/react-svg-pie
2. From a terminal window run `npm ci` to install dependencies.
3. From a terminal window run `npx sb init` as described in the Storybook installation guide.
4. From a terminal window run `npm run storybook`
5. See that all JSX syntax generates warnings
6. See that the Storybook UI says that there are no stories configured

The `npx sb init` command added some example stories to a `stories` folder. I would expect these stories to show up at least in the UI. My own components also have stories defined for them.

I also tried `npx sb init --type react` but this made no difference.

**Comment by shilman** [comment]
This project has a variety of issues:
- It's missing dev dependencies on `react-dom` / `react` / `@material-ui/core`
- It overrides the babel config in a way that's incompatible with storybook

If I add those peer dependencies as dev dependencies and remove the babel config, Storybook loads the CLI-generated stories. There are still problems with the stories in the repo, but I didn't have time to debug them.

**Comment by Bikeman868** [comment]
Thanks for your response.

This is an npm package, and setting dev dependencies is not appropriate. Is there a workaround for that?

I couldn’t find any docs on what Babel configs are compatible or incompatible with Storybook. Presumably I can have a different Babel config just for Storybook? If so, what does it need to look like?

**Reply by shilman** [reply]
why aren't dev dependencies appropriate in an npm package?

**Comment by Bikeman868** [comment]
I was thinking dependencies rather than dev dependencies.

There is no issue with having react in dev dependencies and peer dependencies so I will do that.

Any docs on what works or doesn’t work in the Babel config?
